srcpkgs/sway/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'sway'
 pkgname=sway
 version=1.7
```

Comment:
Can we bump to 1.8rc1 which uses wlroots 0.16 for testing?

Reply:
Done